.changelog/2107.breaking.md

```diff
@@ -0,0 +1,5 @@
+go node: Unite compute, merge, and transaction scheduler roles.
+
+We're removing the separation among registering nodes for the compute, merge, and transaction scheduler roles.
+You now have to register for and enable all or none of these roles, under a new, broadened, and confusing--you're
+welcome--term "compute."
```

Comment:
Yeah this is quite confusing, maybe we should make it so:
* The overall term for all three is "compute".
* The old "compute" is renamed to "executor" (or something, no strong preference here).

Thoughts?

Reply:
ya ok

Reply:
#2525